lib/cartopy/mpl/feature_artist.py

```diff
@@ -227,3 +227,30 @@ def draw(self, renderer, *args, **kwargs):
 
         # n.b. matplotlib.collection.Collection.draw returns None
         return None
+
+
+# Put setters, getters and properties that one would expect on a matplotlib
+# artist.
+def _attach_setters_and_getters(prop):
+    getter_name = 'get_{}'.format(prop)
+    if not getattr(FeatureArtist, getter_name, None):
+        # Attach a getter for the prop.
+        def getter_meth(self):
+            return self._kwargs.get(prop, None)
+        setattr(FeatureArtist, getter_name, getter_meth)
+
+    setter_name = 'set_{}'.format(prop)
+    if not getattr(FeatureArtist, setter_name, None):
+        def setter_meth(self, v):
+            return self._kwargs.__setitem__(prop, v)
```

Comment:
Do you want this to be able to change `facecolor='never'`, or should this use `style.merge`?

Reply:
To be clear, ``facecolor='never'`` is mostly coming from NaturalEarthFeature instances. The kwargs being changed here are on the ``FeatureArtist`` instances. I have no objection to users being able to add their own ``facecolor='never'`` here, but it feels like it isn't as useful as at the NaturalEarthFeature level.

Because the style is being defined in a single dict here we can follow the semantics of matplotlib (facecolor being trumped by color, etc.). Consumers of this dictionary *may* choose to merge these things together (OK, they will **always** choose to merge) but that is an implementation detail down the stack.